src/registrar/forms.py

```diff
@@ -0,0 +1,10 @@
+from django import forms
+
+from .models import UserProfile
+
+class EditProfileForm(forms.ModelForm):
```

Comment:
Docstring.

It can feel redundant to put class documentation on a class which is so obviously self-explanatory, but future maintainers may not be as familiar with Django as you are. It is a kindness to them to include a brief but clear explanation of why a piece of code exists.